docs/prepare-environment/custom-env.txt

```diff
@@ -8,22 +8,22 @@ Determined launches workloads using Docker containers. By default, workloads exe
 Determined-provided container that includes common deep learning libraries and frameworks.
 
 If your model code has additional dependencies, the easiest way to install them is to specify a
-:ref:`startup hook <startup-hooks>`. For more complex dependencies, you can also use a :ref:`custom
-Docker image <custom-docker-images>`.
+:ref:`startup hook <startup-hooks>`. For more complex dependencies, use a :ref:`custom Docker image
+<custom-docker-images>`.
 
-If you're using Determined on Kubernetes, review the supplementary :ref:`Custom Pod Specs
-<custom-pod-specs>` guide.
+If you are using Determined on Kubernetes, review the :ref:`Custom Pod Specs <custom-pod-specs>`
+guide.
 
 .. _environment-variables:
 
 ***********************
  Environment Variables
 ***********************
 
-For both trial runners and commands, Determined allows users to configure the environment variables
-inside the container through the ``environment.environment_variables`` configuration field of the
-:ref:`experiment <experiment-configuration>` or :ref:`task <command-notebook-configuration>` config.
-The format is a list of strings in the format ``NAME=VALUE``:
+For both trial runners and commands, you configure the environment variables inside the container
+using the :ref:`experiment <experiment-configuration>` or :ref:`task
```

Comment:
I really wish we had anchors for the individual config options.

Reply:
As long as the anchors are headings, that's fine. I'll make them headings if they're not and if it makes sense.

Reply:
I'll add that to the wish list. But, it's generally an unmaintainable feature, which is why you only link to headings and even that's problematic. Overall, there's too much linking in the Determined docs but that first needs to be addressed at a higher level by restructuring, then by defining a sane linking model.